### PR TITLE
Change the date to past date for example dag

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
Fixing group 1 - https://github.com/astronomer/astro-sdk/actions/runs/3368325263/jobs/5586745425#step:11:834

### CODE REVIEW

**Review by phanikumv — APPROVED** [review]

File: python-sdk/example_dags/example_datasets.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 # [END dataset_table]
 
 top_animations_table = Table(name="top_animation", conn_id="sqlite_default")
-START_DATE = datetime(2022, 9, 1)
+START_DATE = datetime(2016, 9, 1)
```
Comment (kaxil) [inline python-sdk/example_dags/example_datasets.py:37]: Why was this needed? The date was already in past, Sep 9, 2022?

Comment (utkarsharma2) [reply python-sdk/example_dags/example_datasets.py:37]: Nope, START_DATE was used to initialize dag.

Screenshot 2022-11-01 at 7 28 54 PM

Comment (utkarsharma2) [reply python-sdk/example_dags/example_datasets.py:37]: ahh yes, but weird that fixed the issue in CI. Maybe an intermittent issue
